bin/crew

```diff
@@ -55,7 +56,7 @@ DOC = <<~DOCOPT
     crew postinstall [options] [-v|--verbose] <name> ...
     crew prop [-v|--verbose]
     crew reinstall [options] [-k|--keep] [-s|--source] [-S|--recursive-build] [-v|--verbose] <name> ...
-    crew remove [options] [-v|--verbose] <name> ...
+    crew remove [-v|--verbose] <name> ...
```

Comment:
I'm still foggy on what exactly `[options]` is used for inside docopt.

Reply:
Me too-- it seems safe to remove it so far though, haven't encountered any problems.

Reply:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/6ec31cdc-d404-4f66-a2c5-70110f35efc2)
Looks like we definitely don't need it here.